### PR TITLE
Initial version of SAML example

### PR DESCRIPTION
**Pull request by brian-vendia**
This is the code example to accompany a blog post on using SAML federation to authenticate requests to Share.

### CODE REVIEW

File: share/saml-example/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+### Change to the saml-example Directory
+
+```bash
+cd examples/share/saml-example
+```
+
+Now that we've cloned the repository, [let's set up Auth0 as a SAML identity provider with an Amazon Cognito user pool](create-cup.md).
+
+
+# Creating a
```
Comment (james-vendia) [inline share/saml-example/README.md:46]:
```suggestion
# Creating a simple product catalog
```

File: share/saml-example/create-and-work-with-uni.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## Reading Data from Our DistributionCenter Node
+
+Our Uni will not have any `InventoryItems` in it since we did not populate it with `initState` data. However, we can still issue a valid GraphQL query to confirm authentication works as expected. We will need to make a request of our **DistributionCenter** `httpsUrl` GraphQL endpoint. This can be done using a GUI like the [Altair GraphQL Client](https://altair.sirmuel.design/) or even a command line utility like [curl](https://man7.org/linux/man-pages/man1/curl.1.html). The `Authorization` header must be present regardless of the utility used to make the GraphQL query.
+
+```bash
+% curl ${GRAPHQL_URL} \
+-H 'Content-Type: application/json' \
+-H "Authorization: ${AUTHORIZATION}" \
+--data-binary '{"query": "query listAllProducts {\n  listInventoryItems {\n    InventoryItems {\n      _id\n      itemName\n      itemNumber\n      quantity\n    }\n  }\n}","variables":{}}' \
```
Comment (james-vendia) [inline share/saml-example/create-and-work-with-uni.md:66]: I'd double check this (if you haven't already) against the latest/greatest version of Share.  Not sure how the "Self_" namespace will impact the GQL here, if at all.

**Review by james-vendia — APPROVED** [review]
Aside from double checking against the latest/greatest of Share, I think this looks good to go.

File: share/saml-example/README.md (hunk 1 above)
Comment (brian-vendia) [reply share/saml-example/README.md]: Didn't mean to have this section in the `README.md` file - removing it.

File: share/saml-example/create-and-work-with-uni.md (hunk 2 above)
Comment (brian-vendia) [reply share/saml-example/create-and-work-with-uni.md]: Good call - the queries worked before the push but they needed to be updated.

**Pull request merged**